packages/expo/src/takeSnapshotAsync/Images.web.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  getSourceAndEncodeAsync,
+  formatDataAsUrl,
+  makeIterable,
+  isDataUrl,
+  getMimeTypeFromSource,
+} from './Utils.web';
+
+import { batchProcessAllSourcesAsync } from './ProcessSources.web';
+
+export async function processAllImagesAsync(element: HTMLElement): Promise<void> {
+  if (!(element instanceof HTMLElement)) {
+    return;
+  }
+
+  await processAndMutateNodeBackgroundImageAsync(element);
+
+  if (element instanceof HTMLImageElement) {
+    await loadNewImageAsync(element);
+  } else {
+    const children = makeIterable(element.childNodes);
```

Comment:
```suggestion
    const children = Array.from(element.childNodes);
```

This is basically makeIterable but built in